user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -580,7 +580,8 @@ define([
       showDescriptions: false,
       deletedFiles: "",
 
-      sequenceNumber: 0
+      sequenceNumber: 0,
+      keysSoFar: ''
```

Comment:
Please, use double quotes whenever possible: `""`. This applies to here and all other cases on the changes of this PR.